lib/Onyx.js

```diff
@@ -794,6 +802,36 @@ function notifyCollectionSubscribersOnNextTick(key, value) {
     Promise.resolve().then(() => keysChanged(key, value));
 }
 
+/**
+ * This method waits for prior modifications to complete before running the next modification.
```

Comment:
Please define what a "modification" is in this code comment with an example prior to explaining what the function will do with them.